.github/ISSUE_TEMPLATE/batch-email-requests.yml

```diff
@@ -0,0 +1,79 @@
+name: Batch Email Request Form
+description: Request the CS team to send out an batch email
+title: "[Batch Email]: "
+labels:
+  - "email"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### This issue template can be used to request that the CS team send out an ad hoc email blast when Cal-ITP has noteworthy news to share. The process for preparing an email blast is [detailed in this document](https://docs.google.com/document/d/1FZBgW3vASHcPKmdD38V3zexcBNtyz-jZnjHQcwf4HBU/edit).
+  - type: input
+    id: email-copy
+    attributes:
+      label: Provide a link to the email copy
+      description: The link should be a google doc
+      placeholder: ex https://docs.google.com/
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        _If no, the CS team will wait to take action on this issue until told that the copy is final_
+  - type: checkboxes
+    id: information
+    attributes:
+      label: Confirm that the linked document above provides the following pieces of information
```

Comment:
Can we make this question required as well?

Reply:
that's fine, not a huge deal! Should be important enough that people won't skip it.